Cargo.toml

```diff
@@ -84,6 +85,7 @@ criterion = "0.3"
 [features]
 developer-mode = []
 default = ["developer-mode"]
+monitoring = ["prometheus"]
```

Comment:
Can we just call this feature flag `prometheus`?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1670/commits/43317acdb83474ae49ef9f2b8d2b7aad21080607

Reply:
Ha, will actually roll back
```
Caused by:
  Features and dependencies cannot have the same name: `prometheus`
```

Reply:
maybe `monitoring-prom` ?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1670/commits/1458a8cd4c0285c4458a9312fa7de72f5c33e0c6